ci_framework/playbooks/ceph.yml

```diff
@@ -180,6 +198,16 @@
         name: cifmw_cephadm
         tasks_from: pools
 
+    - name: Deploy RGW
+      ansible.builtin.import_role:
+        name: cifmw_cephadm
+        tasks_from: rgw
+      vars:
+        # public network always exist because is provided by the ceph_spec role
+        cifmw_cephadm_rgw_network: "{{ lookup('ansible.builtin.ini', 'public_network', section='global',  file=cifmw_cephadm_bootstrap_conf) }}"
```

Comment:
Maybe we can set this to a fact and call it earlier?

Reply:
Yeah, we can do that: is there a particular reason to prefer a `set_fact` over a `var` ?

Reply:
I think there's a reason to have it as a fact and not as a var: the first reason I've found is to remove the `storage_network_range` variable that is not required at this point.

Reply:
Done, now this is a fact that replaces the need of hardcoding a `storage_network`.